core/common/src/main/java/com/hilingual/core/common/event/trigger/LocalDialogTrigger.kt

```diff
@@ -13,10 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.hilingual.core.designsystem.event
+package com.hilingual.core.common.event.trigger
 
 import androidx.compose.runtime.staticCompositionLocalOf
 
-val LocalDialogEventProvider = staticCompositionLocalOf<DialogEventProvider> {
+val LocalDialogTrigger = staticCompositionLocalOf<DialogTrigger> {
     error("No DialogEventProvider provided")
```

Comment:
```suggestion
    error("No DialogTrigger provided")
```
p2
사소하긴 하지만..ㅎ 이름 수정한 것처럼 에러 메시지 부분도 같이 수정해주면 나중에 문제 발생했을 때 찾기 쉬울 것 같아요!

Reply:
맞네요ㅎㅎ 수정하겠습니다. 추가로 언제 에러가 발생할까요?

Reply:
@nahy-512 @nhyeonii @Daljyeong 스스로 공부하기 숙제